.github/workflows/macos-linux-windows-pixi.yml

```
@@ -264,12 +263,51 @@ jobs:
         run: |
           pixi run -e test-pixi-build test
 
+  pinocchio-rtsan-build:
+    name: RTSan build - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            name: Ubuntu (x86_64)
+
+          - os: macos-latest
+            name: MacOS (ARM64)
```

```suggestion
        os: [ubuntu-latest, macos-latest]
```

`name` is never used.

---

Using name instead 👍